src/views/Schedule2.vue

```diff
@@ -53,6 +65,23 @@ export default Vue.extend({
         top: item.top * this.ratio + 10 + 'px',
         left: this.width * index + 10 + 'px'
       };
+    },
+    updateNowBarOffset() {
```

Comment:
A unit test in jest would be good, we can merge without to get the feature but in a second pr I would be very happy to see unit test for this method!

Reply:
I added a unit test to assert the value of the offset once the component is mounted. But I could not call this method from the unit test nor spy on it as methods of Vue components are not available in their API or in the test wrapper.

Reply:
Done